tests/processor/test_tasks.py

```diff
@@ -109,31 +111,43 @@ class TestProcessorTasks(MasuTestCase):
 
     fake = faker.Faker()
 
+    def setUp(self):
```

Comment:
Everything done here is quick enough that it's okay to do it for each test, but I just wanted to point out https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUpClass for future reference. You can do a 1 time setup for the whole class for anything that is okay to share between test cases.

Reply:
Totally agree. IMO, the main reason to do it in `setUp()` versus `setUpClass()` is just so that Faker gives us new random values with each test.